src/main/java/com/gmail/goosius/siegewar/SiegeController.java

```diff
@@ -207,16 +205,7 @@ public static boolean loadSiege(Siege siege) {
 		siege.setWarChestAmount(SiegeMetaDataController.getWarChestAmount(town));
 		siege.setTownPlundered(SiegeMetaDataController.townPlundered(town));
 		siege.setTownInvaded(SiegeMetaDataController.townInvaded(town));
-
-		if (SiegeMetaDataController.getStartTime(town) == 0l)
-			return false;
-		siege.setStartTime(SiegeMetaDataController.getStartTime(town));
-
-		if (SiegeMetaDataController.getEndTime(town) == 0l)
-			return false;
-		siege.setScheduledEndTime(SiegeMetaDataController.getEndTime(town));
-
-		siege.setActualEndTime(SiegeMetaDataController.getActualEndTime(town));
+		siege.setNumBattleSessionsCompleted(0);

```

Comment:
Does this 0 get replaced somewhere else?

Reply:
Ah, it doesn't right now, which is definitely an oopise. Will fix

Reply:
Fixed